opal/mca/common/ucx/common_ucx_wpool.c

```diff
@@ -454,7 +499,7 @@ int opal_common_ucx_wpmem_create(opal_common_ucx_ctx_t *ctx,
     }
```

Comment:
1. Move the code from lines 486-499 into the wpool scope (logically belongs there)
2. Lock wpool mutex around it to protect ucp context
3. _comm_ucx_wpmem_map should be in the wpool scope.